platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SignatureTransmitter.java

```diff
@@ -67,8 +67,9 @@ public void transmitSignature(final long round, @NonNull final Signature signatu
         Objects.requireNonNull(signature);
         Objects.requireNonNull(stateHash);
 
-        if (getPlatformStatus.get() != PlatformStatus.ACTIVE) {
-            // Don't bother sending signature transactions if not active.
+        final PlatformStatus platformStatus = getPlatformStatus.get();
+        if (platformStatus != PlatformStatus.ACTIVE && platformStatus != PlatformStatus.MAINTENANCE) {
+            // Don't bother sending signature transactions if not active or during the freeze process.
```

Comment:
This comment is a little confusing. I suggest `Only send transactions if the platform is in ACTIVE or MAINTENANCE state`

Reply:
optional suggestion

Reply:
done